x-pack/metricbeat/module/azure/azure.go

```diff
@@ -17,10 +17,10 @@ import (
 
 // Config options
 type Config struct {
-	ClientID            string           `config:"client_id"    validate:"required"`
+	ClientId            string           `config:"client_id"    validate:"required"`
 	ClientSecret        string           `config:"client_secret" validate:"required"`
-	TenantID            string           `config:"tenant_id" validate:"required"`
-	SubscriptionID      string           `config:"subscription_id" validate:"required"`
+	TenantId            string           `config:"tenant_id" validate:"required"`
+	SubscriptionId      string           `config:"subscription_id" validate:"required"`
```

Comment:
Why these changes? I don't think there is anything wrong with ID

Reply:
there was a mix of `Id` and `ID` in the code so I went with most used for consistency